### ISSUE TITLE
Ch 12, yml changes

### ISSUE THREAD

**Issue by SolidRockProgrammer** [PacktPublishing/-Learn-WinUI-3.0#9]
I'm getting errors related to https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

Also, is this currently valid?
uses: Azure/static-web-apps-deploy@v0.0.1-preview
should it be?
uses: Azure/static-web-apps-deploy@v1

**Comment by alvinashcraft** [comment]
Yes, I think it should be. The preview version is what was available at the time of publication.
